### PR TITLE
Load all pathogenic

### PR DESCRIPTION
**Pull request by moonso**
This PR changes so that scout will load all variants with a clinvar annotation in `['pathogenic', 'likely_pathogenic', 'conflicting_interpretations_of_pathogenecity']`

**How to test**:
There are some new test files to check that the behaviour works:

There are two small VCFs in `tests/fixtures/vcfs/` with two variants in them. One file (`no_clinvar.vcf`) lacks clinvar annotation. Only one of the variants should be loaded (because of rank score). The other file (`clinvar.vcf`) have the same variants but with clinvar annotation. When this file is used, both variants should be loaded.

1. Install this branch
1. Load the case without clinvar and check that only one variant is loaded.
```scout load case --vcf tests/fixtures/vcfs/no_clinvar.vcf --ped tests/fixtures/peds/643594.ped --owner cust000``` 
Check that only one variant is loaded
1. Delete the case: `scout delete case --case-id clinvar`
1. Load case again with the file that have clinvar annotation
```scout load case --vcf tests/fixtures/vcfs/clinvar.vcf --ped tests/fixtures/peds/643594.ped --owner cust000``` 
1. Check that both variants are loaded regardless of rank score.

**Expected outcome**:
All variants with the given clinvar annotations should be loaded

**Review:**
- [x] code approved by
- [x] tests executed by dNil

### CODE REVIEW

**Comment by moonso** [comment]
@dnil it looks like they are there: https://github.com/Clinical-Genomics/scout/pull/1379/files#diff-aca9aca5c69d7db3e505efef30624b6f

These files will not be used by any fixture or automated testing (as for now), they are just there to test the commands in the explanation.

**Comment by dnil** [comment]
Thanks! Found it finally; was looking in the wrong branch there for a bit..

File: scout/parse/variant/variant.py [hunk 1]
```diff
@@ -235,16 +235,7 @@ def parse_variant(variant, case, variant_type='clinical',
...
-        acc = variant.INFO.get('CLNACC')
-    else:
-        acc = variant.INFO.get('CLNVID')
-    clnsig_predictions = parse_clnsig(
-        acc=acc,
-        sig=variant.INFO.get('CLNSIG'),
-        revstat=variant.INFO.get('CLNREVSTAT'),
-        transcripts=parsed_transcripts
-        )
+    clnsig_predictions = parse_clnsig(variant, transcripts=parsed_transcripts)
```
Comment (dnil) [inline scout/parse/variant/variant.py]: :+1:

File: tests/parse/test_parse_clnsig.py [hunk 2]
```diff
@@ -1,55 +1,215 @@
...
 
     for entry in clnsig_annotations:
-        assert entry['accession'] ==  int(variant['info_dict']['CLNACC'])
-        assert entry['value'] in ['Pathogenic', 'Likely pathogenic']
-        assert entry['revstat'] == ', '.join(revstat_groups)
+        assert entry['accession'] ==  int(acc_nr)
+        assert entry['value'] in ['pathogenic', 'likely_pathogenic']
+        assert entry['revstat'] == ','.join(revstat_groups)
+
+def test_parse_complex_clnsig(cyvcf2_variant):
```
Comment (dnil) [inline tests/parse/test_parse_clnsig.py]: :+1:

File: scout/constants/clnsig.py [hunk 3]
```diff
@@ -7,16 +7,43 @@
...
+    'association_not_found': 13,
+    'risk_factor': 12,
+    'protective': 11,
+    'association': 10,
+    'affects': 9,
+    'histocompatibility': 7,
     'benign': 2,
     'likely_benign': 3,
+    'conflicting_interpretations_of_pathogenicity':8,
+    'uncertain_significance': 0,
```
[the remark this answers is not in the document]
Comment (moonso) [reply scout/constants/clnsig.py]: Hehe yes not the optimal choice. We'll see what happens in the future for clinvar...

File: scout/constants/clnsig.py [hunk 4]
```diff
@@ -7,16 +7,43 @@
...
     'likely_pathogenic': 4,
     'pathogenic': 5,
-    'drug_response': 6,
-    'histocompatibility': 7,
-    'other': 255
+    0: 0,
```
[the remark this answers is not in the document]
Comment (moonso) [reply scout/constants/clnsig.py]: Sure

**Pull request merged**